api/src/main/java/org/pmiops/workbench/monitoring/MonitoringServiceImpl.java

```diff
@@ -73,22 +74,31 @@ public void recordValue(OpenCensusStatsViewInfo viewInfo, Number value) {
   }
 
   /**
-   * Record multiple values at once, indexed by keys in the StatsViewProperties enum. This should
-   * save calls to Stackdriver.
+   * Record multiple values at once. An attachment map allows associating these measurements with
```

Comment:
nit: bump the method description up to the interface level, for consistency with the nice docs added for recordEvent.